OpenBLAS.spec

```diff
@@ -10,12 +10,12 @@ Source: https://github.com/xianyi/OpenBLAS/archive/v%{realversion}.tar.gz
 
 # PRESCOTT is a generic x86-64 target https://github.com/xianyi/OpenBLAS/issues/685 
 %ifarch x86_64
-make FC=gfortran BINARY=64 TARGET=PENRYN NUM_THREADS=256 DYNAMIC_ARCH=1
+make FC=gfortran BINARY=64 TARGET=CORE2 NUM_THREADS=256 DYNAMIC_ARCH=0
```

Comment:
what does the ```NUM_THREADS``` actually correspond to in reality at run time?
Was the speedup seen in tests of deepAK8 real or a simple parallelization on a quiet node?

Reply:
@slava77 What affects the number of threads used at runtime is the `OPENBLAS_NUM_THREADS` environment variable which is set to 1 in https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_11_0_X/gcc700/OpenBLAS-toolfile.spec#L19. `NUM_THREADS` is the maximum allowed number of threads and is used for resource allocation etc. I explicitly checked the CPU usage when running the tests and it was not going beyond 100%:)

Reply:
And `gslblas` is known to be not a very high-performance BLAS routine, so the speed-up we saw after switching to `OpenBLAS` should not be surprising at all.

Reply:
thanks for clarifications